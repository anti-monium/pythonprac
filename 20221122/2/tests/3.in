«äîá¶îå óòðî è ªõîð½îøåãî äíÿ 
âñåì äîáðûì è ñî§îòâåòßñòâåííî õîðîøèì† ëþäÿ¦ì…


