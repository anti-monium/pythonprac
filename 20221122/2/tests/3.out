«доб¶ое утро и ЄхорЅошего дня 
всем добрым и со§ответЯственно хорошим? людя¦м?
